resolwe_bio/processes/WGBS/mcall.yml

```diff
@@ -0,0 +1,138 @@
+# =====================================
+# The methylation ratio calling module
+# =====================================
+---
+
+- slug: mcall
+  name: The methylation ratio calling module
+  requirements:
+    expression-engine: jinja
+  data_name: "mcall"
+  version: 0.0.1
+  type: data:wgbs:mcall
+  category: analyses:wgbs
+  persistence: CACHED
+  description: >
+    The methylation ratio calling module accepts alignment from BSMAP,
+    and then performs methylation ratio calling on every covered
+    cytosine for both WATSON and CRICK genomes.
+  input:
+    - name: genome
+      label: Reference genome
+      type: data:genome:fasta
+    - name: genome_identifier
+      label: The UCSC Genome Browser identifier
+      type: basic:string
+      choices:
+        - label: hg19
+          value: hg19
+        - label: hg18
+          value: hg18
+        - label: mm8
+          value: mm8
+        - label: mm9
+          value: mm9
+        - label: mm10
+          value: mm10
+    - name: bam
+      label: Bam
+      type: list:data:alignment:bam
```

Comment:
Shouldn't this be list:data:alignment:bam:bsmap? Or is the mcall general enough to accept BAMs from other tools?